src/Razor/src/Microsoft.CodeAnalysis.Razor.Workspaces/Completion/CompletionSortTextHelper.cs

```diff
@@ -0,0 +1,12 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT license. See License.txt in the project root for license information.
+
+namespace Microsoft.CodeAnalysis.Razor.Completion
+{
+    internal static class CompletionSortTextHelper
+    {
+        public static string DefaultSortPriority => null;
+
+        public static string HighSortPriority => " ";
```

Comment:
This is amazing.

The LSP spec does specify that when null, it will use label instead of sort text, but its not clear that it falls back to label if two sort text's are the same. I assume thats what is happening here?

Reply:
lol lovely aint it?

So ya the assumption I'm making in regards to sort order is that a client will need to fallback to something to sort and that the completion item `label` is the technnnnically the only reasonable thing to fallback to 😄

Reply:
I wonder if a comment in the code might be helpful here? On first glance, I think I would be confused on why the two strings are defined the way they are if you hadn't explained it

Reply:
Ya it definitely would be helpful, great idea!